BUILDING.md

```diff
@@ -97,7 +97,8 @@ More Developer Tools...`. This step will install `clang`, `clang++`, and
 * After building, you may want to setup [firewall rules](tools/macosx-firewall.sh)
 to avoid popups asking to accept incoming network connections when running tests:
 
-If the path to your build directory contains a space, the build will likely fail.
+If the path to your build directory contains a space or a non-ASCII character, the
+build will likely fail.
```

Comment:
I would not put this into the unix section, experiences from recent Code & Learn events have shown that we are not usually that far away from this working. 

The only issue I know of is fixed by v8/v8@744b49ef0d1b72de2f55994c38ff0d4dea6a2d10, which comes with the V8 6.4 upgrade (https://github.com/nodejs/node/pull/17489).